OpenRA.Game/Traits/Player/Shroud.cs

```diff
@@ -54,12 +54,15 @@ public class ShroudInfo : TraitInfo, ILobbyOptions
 		[Desc("Display order for the explore map checkbox in the lobby.")]
 		public readonly int ExploredMapCheckboxDisplayOrder = 0;
 
+		[Desc("Options category in which to display the <e.g. 'developer mode checkbox' or 'DefaultCash option'> in the lobby.")]
+		public readonly string Category = null;
```

Comment:
Sorry to keep going on about this, but can we please rename these to match the other checkbox fields? So `ExploredMapCheckboxCategory` etc across the different traits.